siac/hostdbcmd.go

```diff
@@ -117,15 +120,30 @@ func hostdbcmd() {
 				uptimeRatio = float64(uptime) / float64(uptime+downtime)
 			}
 
+			scanHistStr := ""
+			displayScans := host.ScanHistory
+			if len(host.ScanHistory) > scanHistoryLen {
+				displayScans = host.ScanHistory[len(host.ScanHistory)-scanHistoryLen:]
+			}
+			for _, scan := range displayScans {
+				if scan.Success {
+					scanHistStr += "1"
+				} else {
+					scanHistStr += "0"
+				}
+			}
```

Comment:
I suppose it's fine for now, but I don't think 1 and 0 are intuitive or distinct. We'll see what users say.
Another issue is that there's no hint as to which end is the more recent end, or the timescale